client_files/import-wiki.sh

```diff
@@ -66,19 +66,46 @@ meza1_root="/var/www/meza1"
 wiki_root="$meza1_root/htdocs/wiki"
 smw_root="$wiki_root/extensions/SemanticMediaWiki"
 wiki_sql_file="$meza1_root/wiki.sql"
-wiki_images="$meza1_root/images.tar.gz"
-cd "$meza1_root/htdocs/wiki"
+wiki_images="$meza1_root/images"
```

Comment:
After a thorough review, I see how this works. At first, though, the variable name "wiki_images" was misleading. I immediately thought it would be the images directory in the wiki directory ($wiki_root/images). It's actually a temp folder for the images being imported. What do you think about changing the name to something like "imported_images"? It might then make sense to change "wiki_sql_file" to "imported_sql_file".